content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -436,192 +618,274 @@ Resources are deleted in two phases: 1) finalization, and 2) removal.
 }
 ```
 
-When a client first deletes a resource, the `.metadata.deletionTimestamp` is set to the current time.
+When a client first **delete**s a resource, the `.metadata.deletionTimestamp` is set to the current time.
```

Comment:
Q:  The original sentence used the verb "deletes" in a general way. When the verb is formatted, does this imply that the client is making a request in a special way?

Reply:
The client is telling the API server to perform deletion.

Reply:
OK, yes. I looked at the preview. The partial formatting of the verb does not look as nice. It's okay, but may look better if you format the entire string.